packages/components/src/InputNumber/InputNumber.mdx

```diff
@@ -0,0 +1,94 @@
+---
+name: InputNumber
+menu: Components
+route: /components/input-number
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { InputNumber } from ".";
+import { Text } from "../Text";
+import { useState } from "react";
+
+# InputNumber
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Input number is used in forms that accept numbers as an answer.
+
+## Simple Example
+
+<Playground>
+  {() => {
+    const [value, setValue] = useState(1234567890);
+    const handleChange = newValue => {
+      setValue(newValue);
+    };
+    return (
+      <InputNumber value={value} onChange={handleChange} placeholder="Age" />
+    );
+  }}
+</Playground>
+
+## States
+
+### Disabled
+
+<Playground>
+  <InputNumber
+    placeholder="SIN Number"
+    value={1234}
+    disabled={true}
+    name="sin"
+  />
+</Playground>
+
+### Read only
+
+<Playground>
+  <InputNumber
+    placeholder="Your pin number"
+    value={12345}
+    readonly={true}
+    name="pinNumber"
+  />
+</Playground>
+
+### Invalid
+
+<Playground>
+  <InputNumber
+    placeholder="Give a whole number"
+    defaultValue={1.1}
+    invalid={true}
+    name="wholeNumber"
+  />
+</Playground>
+
+## Sizes
+
+<Playground>
+  <InputNumber name="smallField" size="small" placeholder="Duration" />
+  <InputNumber name="largeField" size="large" placeholder="Your username" />
+</Playground>
+
+## Inline
+
+<Playground>
+  <Text>
+    Follow-up after
+    <InputNumber
+      name="durationInDays"
+      value={2}
+      size="small"
+      inline={true}
+      maxLength={4}
+      align="center"
+    />
+    days
+  </Text>
+</Playground>
+
+## Properties
+
+<Props of={InputNumber} />
```

Comment:
For this, it always puts undefined as an acceptable value. Is there a way we can set a default to ensure this this is the expected type? Or are we okay with undefined?

Reply:
Talked in person, we're okay with undefined because we has no control over it... yet